community_guidelines.md

```diff
@@ -15,17 +15,24 @@
 **SELF-PROMOTION**
 * We value your individual journey and the projects you’re working on. Sharing your projects allows the community to assist you.
 * Submit requests for advice on #career-advice. For project promotion, you can post to #brags or #random.
-* If you are a recruiter, please be mindful about not soliciting or spamming members. You may post your positions on #job-board.
-* For code schools, please post all solicitations *ONLY* on #code-schools as this is in our Code of Conduct.
+* The community generally frowns on unsolicited advertising. In general, our guidelines are that you:
+1. Ask permission of the channel before posting the solicitation or information
+2. State clear intent for the purpose
+3. Listen for the response of your peers in channel and act accordingly
+* There are specific channels, such as #job-board and #code-schools, where advertising and recruiting is acceptable.
+* Some geographically-focused channels have a higher tolerance for specific kinds of commercial activity focused on events based in that region. Again, read the room.
+* Commercial requests via direct messages (“DM”) without prior and/or obvious consent from the receiver are prohibited.
+* Please contact moderators@operationcode.org if you have more specific questions.
 
 **ACT PROFESSIONALLY**
-* Operation Code is a community of professionals. Before you post, please think about your content and be inclusive.
-* Operation Code is not Reddit, or a platform for posts that instigate others or incites violence, political ideations or affiliations.
+* Operation Code is a community of professionals. Before you post, please think about your content and be inclusive. 
+* Don’t be a bystander, be a leader. Role model respectful behaviour, but also help to address disrespect when you see it.
```

Comment:
`Model respectful behavior` rather than `Role model respectful behaviour`.

More substantively: this runs the risk of implicitly encouraging vigilante community moderation.

Reply:
Hmm. Do you think we might be better off removing that bullet, or is there a better way to phrase it?